gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/web/internal/configureWebApplication.kt

```diff
@@ -60,6 +58,99 @@ internal fun Project.configureWeb(
     }
 }
 
+private fun Project.introduceComposeWebCompatibilityTask(targets: Collection<KotlinJsIrTarget>) {
```

Comment:
There are some gradle plugin tests in the project.

Would you like to add some tests?
For example:
- a task is created when the condition match and it's not created otherwise
- Run the task and check the output directory exists. And the output directory contains the must-have files (index.html, .js file, wasm file, etc)